dirsrvtests/tests/suites/ds_logs/ds_logs_test.py

```diff
@@ -735,49 +732,6 @@ def test_access_log_truncated_search_message(topology_st, clean_access_logs):
     assert not topo.ds_access_log.match(r'.*cn500.*')
 
 
-
```

Comment:
> There are errors related to `cn=config` access on instance teardown, because it's done under non-DM user (lib389 reuses the open connection from the test).

Sounds like teardown should re-bind to be sure it's DM to avoid that class of problems.

Reply:
Restored test and skipped if newer than 1.4.3